OLE.php

```diff
@@ -133,68 +143,86 @@ function read($file)
         if (!$fh) {
             return $this->raiseError("Can't open file $file");
         }
-        $this->_file_handle = $fh;
 
-        $signature = fread($fh, 8);
-        if ("\xD0\xCF\x11\xE0\xA1\xB1\x1A\xE1" != $signature) {
+        return $this->readStream($fh);
+    }
+
+    /**
+     * Reads an OLE container from the contents of the stream given.
+     *
+     * @access public
+     * @param resource $stream
+     * @return mixed true on success, PEAR_Error on failure
+     */
+    function readStream($stream)
+    {
+        $this->_file_handle = $stream;
+
+        $signature = fread($stream, 8);
+        if (OLE_CFB_SIGNATURE != $signature) {
             return $this->raiseError("File doesn't seem to be an OLE container.");
         }
-        fseek($fh, 28);
-        if (fread($fh, 2) != "\xFE\xFF") {
+        fseek($stream, 28);
```

Comment:
I wish this didn't change `$fh` to `$stream` to keep the git blame aligned. This is nitpicking, truly.

Reply:
Not sure, if I understand it. Should I change the $stream in the readStream() function to $fh?

Reply:
If you can do that, that'd be great. This way `git blame` won't show you as the author of lines where `$fh` was changed to `$stream`, instead pointing at earlier commits.

Reply:
Changed as you requested. Thank you.

Reply:
Excellent! And I just made a release with this fix.